### PR TITLE
add default groups for rook

### PR DESCRIPTION
**Pull request by b1-lender**

### CODE REVIEW

**Comment by berendt** [comment]
Groups must be sorted alphabetically.

File: inventory/50-rook [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+[rook-mds:children]
+ceph-control
+
+[rook-mgr:children]
+ceph-control
+
+[rook-mon:children]
+ceph-control
+
+[rool-osd:children]
```
Comment (berendt) [inline inventory/50-rook:13]: rool should be rook.

**Review by berendt — CHANGES_REQUESTED** [review]
Typo in one group

File: inventory/50-rook (hunk 1 above)
Comment (b1-lender) [reply inventory/50-rook]: fixed

**Comment by berendt** [comment]
Recheck

**Pull request merged**